# Cloud manager
enabled: false               # Use this do disbale/enable the cloud manager (useful to debug navigation)
safe_delay: 1.0             # Wait this period for possible message lag (e.g. from sensor latency or navigation processing)
window_size: 60.0
save_clouds: true

# Processing parameters
min_elevation: -1
max_elevation: 4
min_intensity: 0

grid_size: 1.0            # Down sample point cloud to grid_size*grid_size tiles (in xy-plane)
smoothing_radius: 2.0     # Elevation is smoothed withing smoothing radius
deformation_radius: 5.0   # Deformation is calculated for each point by searchin in deformation_radius

point_buffer_size: 10.0
point_interval: 5.0e-6
shift_cloud: true

min_dist: 20
max_dist: 100

lidar_queue_size: 100
pose_queue_size: 10

save_stats: true